content/en/docs/tasks/security/webhook/index.md

```diff
@@ -0,0 +1,243 @@
+---
```

Comment:
Is this really a security topic? Seems more like something that belongs in the setup portion of the ops guide.

Reply:
Done: moved it to be under the content/en/docs/setup/install/webhook directory.

Reply:
Moved back to security based on the reviewer's comments in https://github.com/istio/istio.io/pull/5285.